client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/utils.ts

```diff
@@ -1,14 +1,26 @@
+import { addQueryArgs } from '@wordpress/url';
+
 const sourceSiteId = 174455321; // dotcompatterns
-const patternPreviewUrl =
-	'https://public-api.wordpress.com/wpcom/v2/block-previews/pattern?stylesheet=pub/blank-canvas&pattern_id=';
+const stylesheet = 'pub/blank-canvas-blocks';
+const publicApiUrl = 'https://public-api.wordpress.com';
+const patternPreviewUrl = publicApiUrl + '/wpcom/v2/block-previews/pattern';
+
+const getPatternPreviewUrl = ( id: number, language: string ) => {
+	const pattern_id = `${ id }-${ sourceSiteId }`;
 
-const getPatternPreviewUrl = ( id: number ) => `${ patternPreviewUrl }${ id }-${ sourceSiteId }`;
+	return addQueryArgs( patternPreviewUrl, {
+		stylesheet,
+		preview_auto_height: true,
```

Comment:
Since D87232-code is abandoned, we don't need to pass `preview_auto_height` anymore, right?